tensorflow/compiler/tf2tensorrt/trt_convert_api.cc

```diff
@@ -42,10 +46,29 @@ namespace tensorflow {
 namespace tensorrt {
 namespace {
 
+struct ClusterDestroyer {
+  void operator()(grappler::Cluster* c) {
+    if (c) c->Shutdown();
```

Comment:
this returns a Status. How would you want to use the return value?

Reply:
Sorry I have missed it. If an error occurs, then we should return a conversion error. But this is not possible with the current ClusterDestroyer, so I have abandoned this functor, and call shutdown directly. Let me know if you have a better suggestion.